iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -97,7 +96,7 @@ class ConversationListViewController: ViewController,
 
         self.view.addSubview(self.collectionView)
         self.collectionView.showsVerticalScrollIndicator = false
-        self.collectionView.delegate = self
+        self.collectionView.conversationLayout.delegate = self
```

Comment:
The conversation layout has delegate functions to let us know that the centered conversation has changed.